Gemfile

```diff
@@ -40,3 +40,30 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'rest-client'
 gem 'rspec-rails'
+
+#
+# Custom Gemfile modifications
+#
+# To develop a gem locally and override its source to a checked out repo
+#   you can use this helper method in Gemfile.dev.rb e.g.
```

Comment:
Gemfile.dev.rb should probably be bundler.d/*.rb

Reply:
yeah, done